{
  "statusCode": "400",
  "status": "ERROR",
  "message": "Ocurri√≥ un error inesperado.",
  "data": {
    "errorDetails": {
      "code": "400",
      "fields": {
        "exceptionMessage": "400 Bad Request: \"{\"errorCode\":\"invalid_argument_format\",\"errorMessage\":\"\",\"parameters\":null}\"",
        "exceptionType": "BadRequest"
      }
    }
  },
  "timestamp": "2025-01-12T20:28:53.897983-05:00",
  "traceId": "generated-fallback-transactionId"
}